modules/oe_theme_helper/src/Plugin/PageHeaderMetadata/EntityCanonicalRoutePage.php

```diff
@@ -18,8 +18,9 @@
  *   id = "entity_canonical_route",
  *   label = @Translation("Default entity metadata extractor")
  * )
- *
- * @deprecated in 1.x, will be removed in 2.0.
+ * phpcs:disable Drupal.Commenting.Deprecated
```

Comment:
As discussed, the following solution is recommended.
https://github.com/openeuropa/oe_theme/blob/2.x/modules/oe_theme_helper/src/Plugin/PageHeaderMetadata/EntityCanonicalRoutePage.php#L22

So you can rebase the PR and remove this change.